autobuild/common.py

```diff
@@ -326,12 +326,12 @@ def compute_hash(path: str, hash: Callable[[], hashlib._Hash]):
     Compute a hash for a file effeciently by streaming it into the hash algorithm
     """
     h = hash()
-    b = bytearray(128*1024)
-    mv = memoryview(b)
     try:
         with open(path, 'rb') as f:
-            while n := f.readinto(mv):
-                h.update(mv[:n])
+            chunk = f.read(8192)
+            while chunk:
+                h.update(chunk)
+                chunk = f.read(8192)
```

Comment:
I'm good with this change, but I do notice the chunk size changed from 128K to 8K -- intentional?

Reply:
Haven't bothered to benchmark, but went with smaller/common since it's no longer using a memoryview. I wasn't interested in playing golf and wanted to KISS.